app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ParkingModel.java

```diff
@@ -2,4 +2,12 @@
 
 import com.mobile.tandil.javabaseproject.mvp.contract.ParkingContract;
 
-public class ParkingModel implements ParkingContract.Model {}
+public class ParkingModel implements ParkingContract.Model {
+
+    private int parkingAvailable = 30;
+
+    @Override
+    public int getParkingAvailable() {
```

Comment:
this should get the parkingAvailable value from the parameter

Reply:
fixed